src/commands/run-tests.yml

```diff
@@ -23,6 +23,19 @@ parameters:
       semicolon-separated list.
     type: string
     default: ''
+  select-by-folder:
+    description: >
+      Location of the folder containing test code, relative to the project root folder. The
+      specified folder and its subfolders are added to the test suite to run.To filter multiple test folders
+      You can specify multiple folders using a colon-separated or a semicolon-separated list.
+    type: string
+    default: ''
+  select-by-tag:
+    description: >
+      Specify the test Tag name. Test which are tagged with specified name will be added to test suite and
```

Comment:
Test tag used to select test suite elements. To generate a test suite, MATLAB uses only the test elements with the specified tag.

Reply:
Sounds good to me. changed it